test/nodeMock/workAllocation/mockData.js

```diff
@@ -73,6 +73,56 @@ class WorkAllocationMockData {
         return tasks;
     }
 
+    getCaseList(count, actions) {
+        const caseActions = actions ? actions : [
+            { "id": "reassign", "title": "Reassign task" },
+            { "id": "unclaim", "title": "Release this task" }
+        ];
+        const cases = [];
+        for (let i = 0; i < count; i++) {
+            const taskDueDate = `"2021-02-16T18:58:48.987+0000"`;
```

Comment:
Not entirely sure what the backticks are doing here. 😂

Reply:
It's also not being used from what I can see...

Reply:
Unused, so removed.